media/css/readthedocs-doc-embed.css

```diff
@@ -24,6 +24,10 @@
     margin: 0;
 }
 
+.rtd-current-item {
```

Comment:
noticed we are using `rst-` for the other selectors, not sure why, so I went with `rtd-`

Reply:
I think the CSS for the footer is defined here: https://github.com/readthedocs/readthedocs.org/blob/master/readthedocs/core/static/core/css/badge_only.css

Which is built from here: https://github.com/readthedocs/sphinx_rtd_theme/blob/master/src/sass/badge_only.sass

But we should fix that and include this CSS as part of RTD, but we should put the class in the right place.

Reply:
Done https://github.com/readthedocs/sphinx_rtd_theme/pull/1148, should I just copy that file here? or is it done with a script?

Reply:
...no idea -- looks like it hasn't changed since 2018 :D

Reply:
I'm sure there was a script at some point, but just manually copying it is probably fine.

Reply:
Hmm, I only see this file being included in  our docs https://assets.readthedocs.org/static/css/readthedocs-doc-embed.css

https://github.com/readthedocs/readthedocs-sphinx-ext/blob/f47a64adbb8446238d36c1908ec62694fd7a5dc5/readthedocs_ext/readthedocs.py#L99-L112

https://github.com/readthedocs/readthedocs-sphinx-ext/blob/f47a64adbb8446238d36c1908ec62694fd7a5dc5/readthedocs_ext/readthedocs.py#L174-L174

https://github.com/readthedocs/readthedocs-sphinx-ext/blob/f47a64adbb8446238d36c1908ec62694fd7a5dc5/readthedocs_ext/_templates/readthedocs-insert.html.tmpl#L21-L21

In a mkdocs project both files are included (https://assets.readthedocs.org/static/css/badge_only.css)

https://rtd-mkdocs-test-project.readthedocs.io/en/readthedocs-1.0/ 

https://github.com/readthedocs/readthedocs.org/blob/f4c0a21c24a6777007c588b3185ffb761d9aebc1/readthedocs/doc_builder/backends/mkdocs.py#L169-L172

So, looks like we need to include that change here after all p:

Reply:
@stsewd badge_only is included on non-RTD theme docs, eg. https://pip.pypa.io/en/stable/

Reply:
Created https://github.com/readthedocs/readthedocs.org/issues/8247